src/storage/engine/metrics.rs

```diff
@@ -11,18 +11,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use prometheus::{exponential_buckets, HistogramVec, IntCounterVec};
+use prometheus::exponential_buckets;
+use prometheus_static_metric::*;
+
+use storage::ErrorHeaderKind;
+
+make_static_metric! {
+    pub label_enum RequestStatusKind {
+        all,
```

Comment:
I am surprised clippy does not complain about these being lowercase.

Reply:
Ha. That's because the macro will add a `#[allow]` attribute.